os/se/ameba/security_ameba_wrapper.c

```diff
@@ -399,7 +419,8 @@ int ameba_hal_set_key(hal_key_type mode, uint32_t key_idx, hal_data *key, hal_da
 	if (prikey != NULL) {	/* If private key is pass in */
 		return HAL_NOT_SUPPORTED;
 	}
-	if (key_idx > KEY_STORAGE_INDEX_MAX) {	/* Index Out of Range */
+	key_idx = rtl_index_convt(key_idx);
```

Comment:
is key_idx returned to calling function?
It is passed as argument and calculating value here

Reply:
Yes, it is return to calling function after convert the index.